packages/tools/devtools/devtools-view/src/TelemetryUtils.ts

```diff
@@ -44,6 +45,8 @@ export class ConsoleVerboseLogger implements ITelemetryBaseLogger {
 	public send(event: ITelemetryBaseEvent): void {
 		// Deliberately using console.debug() instead of console.log() so the events are only shown when the console's
 		// verobsity level is set to "Verbose".
+		event.PKG_VERSION = pkgVersion;
```

Comment:
Added `PKG_VERSION` as a separate field in the telemetry `ITelemetryProperties`. Would like to know other folks' thoughts on this @alexvy86. @WayneFerrao, I think if this change gets applied, it'd be easier for your next task that is similar to this [4995](https://dev.azure.com/fluidframework/internal/_sprints/taskboard/Team%20Rohan/internal/Team%20Rohan%20-%202023.J?workitem=4995)?